### PR TITLE
Request to review Invest section features postlogin

### PR DESCRIPTION
**Pull request by sayantanigoswami1996**

### CODE REVIEW

File: Resources/Utils/DesktopWeb/Invest/DigitalGold.robot [hunk 1]
```diff
@@ -98,58 +133,69 @@ Verify PreLogin Digital Gold Landing Page
...
+    Verify Page Contains Element  ${KU_W_dg_paymentModeSubText}
+    Verify Page Contains Element  ${KU_W_dg_paymentBreakup}
+    Verify Page Contains Element  ${KU_W_dg_paymentBreakupGST}
+    Verify Page Contains Element  ${KU_W_dg_GSTText}
+    Verify Page Contains Element  ${KU_W_dg_payableText}
+    Verify Page Contains Element  ${KU_W_dg_totalPayable}
+    Verify Page Contains Element  ${KU_W_dg_paymentCrossIcon}
+    Verify Page Contains Element  ${KU_W_dg_paymentTimer}
+    Verify Page Contains Element  ${KU_W_dg_paymentTimerText}
+    Sleep  305s
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/Invest/DigitalGold.robot:185]: Is this expected? Or can't we increase the timeout in the next line's wait**?

File: Resources/Utils/DesktopWeb/Invest/MutualFunds.robot [hunk 2]
```diff
@@ -15,25 +24,24 @@ Verify PreLogin Mutual Fund Landing Page
         Run Keyword If    ${mutualFundName} == ['Tata Digital India Dividend Reinvest Direct Plan']  Click Element  ${KU_W_invest_mf_growthDividendButton}
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/Invest/MutualFunds.robot:24]: Any specific reason why the expected context is hard-coded?

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_FD.robot [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+    Verify Page Contains Element  ${KU_W_FD_bajajDefaultVal}
+    Wait And Click  ${KU_W_FD_bajajDefaultVal}
+    Verify Page Contains Element  ${KU_W_FD_bajajPopupTitle}
+    Verify Page Contains Element  ${KU_W_FD_defaultCheckedBox}
+    Verify Page Contains Element  ${KU_W_FD_noTitle}
+    Verify Page Contains Element  ${KU_W_FD_RDLabel}
+    Verify Page Contains Element  ${KU_W_FD_BFLLabel}
+    Verify Page Contains Element  ${KU_W_FD_shareholderLabel} 
+    Verify Page Contains Element  ${KU_W_FD_employeeLabel}
+    ${checkBoxCount} =  Get Element Count  xpath=//img[@class='b-check-box__img k-asc']
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_FD.robot:166]: Please move the web element to the locator file.

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_MF.robot [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    Verify In Process Page After Placing Order
+
+
+Verify Buying Of SIP
+    Search For MF 
+    Sleep  1s
+    # Graph Validation with Nifty
+    Verify Page Contains Element  ${KU_W_invest_mf_V/STitle}
+    Wait And Click  ${KU_W_invest_mf_dropdownIcon} 
+    ${niftyGraphCount} =  Get Element Count  xpath=//div[@class='b-mf-sector-filter__option']
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_MF.robot:63]: Please move all the web element to the locator file.

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_MF.robot [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+
+Search For MF 
+    @{mutualFundName} =  Get Json Values  $.MutualFunds.f2  Resources/TestData/MutualFunds.json 
+    Log To Console  ${mutualFundName}
+    Search Fund and Verify  ${mutualFundName} 
+    Sleep  1s
+    Wait And Click  ${KU_W_invest_mf_fundName}
+
+
+Verify Filter Selection For MF
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_MF.robot:208]: Please move all the web element to the locator file.

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_Stocks.robot [hunk 6]
```diff
@@ -0,0 +1,36 @@
+*** Keywords ***
+
+Verify PostLogin Stocks Menu Navigation
+    Verify PreLogin Stocks Landing Page
+    Wait And Click  ${KU_W_stocks}
+    Verify Import Now Banner  ${e_invest_stocks_bannerText}  ${KU_W_postlogin_stocks_importStocksPopup}  ${e_postlogin_stocks_importPopup}
+    Verify Filter Selection For Stocks
+    
+Verify Filter Selection For Stocks
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_Stocks.robot:9]: Please move all the web element to the locator file.

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_USStocks.robot [hunk 7]
```diff
@@ -0,0 +1,37 @@
+*** Keywords ***
+
+Verify PostLogin US Stocks Menu Navigation
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_USStocks.robot:3]: Please move all the web element to the locator file.

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_USStocks.robot [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    Verify PreLogin US Stocks Landing Page
+    Wait And Click  ${KU_W_investLink}
+    Wait And Click  ${KU_W_USStocks}
+    Wait And Click  ${KU_W_invest_USStocks_exploreBtn}
+    Verify Import Now Banner  ${e_invest_USStocks_bannerText}  ${KU_W_postlogin_USStocks_linkYourVestedAcc}  ${e_postlogin_USStocks_linkYourVestedAcc}
+    Verify Filter Selection For USStocks
+
+ Verify Filter Selection For USStocks
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_USStocks.robot:11]: Please move all the web element to the locator file.

File: Resources/Utils/DesktopWeb/Invest/DigitalGold.robot (hunk 1 above)
Comment (sayantanigoswami1996) [reply Resources/Utils/DesktopWeb/Invest/DigitalGold.robot:185]: I will check into this

File: Resources/Utils/DesktopWeb/Invest/MutualFunds.robot (hunk 2 above)
Comment (sayantanigoswami1996) [reply Resources/Utils/DesktopWeb/Invest/MutualFunds.robot:24]: We can remove it

File: Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_FD.robot (hunk 3 above)
Comment (sayantanigoswami1996) [reply Resources/Utils/DesktopWeb/PostLoginFlows/InvestExplore/Po_FD.robot:166]: Okay

**Pull request merged**